configure.ac

```diff
@@ -70,6 +70,19 @@ GX_C__GENERIC
 AS_IF([test $gx_cv_c__Generic = no],
   AC_MSG_ERROR([The C compiler does not support the generic selection C-11 standard.  Please use a C-11 compliant compiler.])
 )
+# openacc support, simple check for working (nvc)openacc flag and allows for disabling if desired
+# TODO this should really be added as an m4 macro, surprised there wasn't a existing one in the archives
+AC_ARG_WITH([openacc],
+  [AS_HELP_STRING([--with-openacc],
+    [Builds with openacc flags if available (default no)])])
+AS_IF([test ${with_openacc:-no} = yes],
+  [with_openacc=yes],
+  [with_openacc=no])
+AS_IF([test ${with_openacc} = yes],
+  [AX_CHECK_COMPILE_FLAG( [-acc],
+       [CFLAGS="-acc -tp haswell -gpu=cc70 -Minfo -Mnoinline"],
```

Comment:
I think -Minfo=accel will give useful info about openacc compilation

Reply:
@mlee03 @rem1776 
The flags I was using for compiling for the gpu are:
 O2 -g -traceback -acc -ta=nvidia:managed -Minfo -Mbounds  -Mnoinline  

I think we may need to compare with the section "-tp haswell -gpu=cc70"

Reply:
The -tp is for the cpu architecture on the gpu box, not sure if thats technically needed.

-ta has been deprecated, -gpu is the new version with cc70 being the 'compute capability' that's on the tesla. After looking at the options a bit more there is also a `managed` option that can be added to the gpu flag, it turns on using CUDA managed memory. I'm not sure what that means entirely since we're not using cuda mallocs or anything like that in this code, but i'll test it out.

I'll definitely add in these:  `-Minfo=accel -O2 -g`

Reply:
I don't think we'll use the `managed` option after we optimize all the data transfers